backend/sending_data/sendData.py

```diff
@@ -0,0 +1,25 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+from dotenv import load_dotenv
+import os
+import json
+
+def run(data):
+    load_dotenv()
+    URL = os.getenv("FIREBASE_REALTIME_DATABASE_URL") 
+    with open("keys.json") as f:
+        SECRET = json.load(f)
+
+    ### Firebase Connection
+    cred = credentials.Certificate(SECRET)
+    firebase_admin.initialize_app(cred, {
+        "databaseURL": URL
+    })
+
+    ref = db.reference("/")
+    data = list(data)
+    ref.set({
+        "data": data
+    })
+    print("Sent data to firebase!")

```

Comment:
### Return value of `run(data)`

If we don't explicitly state `return` in Python's method, Python will automatically return a default value. However, in case of collaboration, we would like to make sure that the `run(data)` is working well from `main.py` where `run(data)` will be called. 

Please add a `return` statement returning data itself and confirmation message like "data successfully sent".